images/kubekins-e2e/kops-e2e-runner.sh

```diff
@@ -69,25 +69,6 @@ if [[ -z "${EXTERNAL_IP}" ]]; then
 fi
 e2e_args+=(--kops-admin-access="${EXTERNAL_IP}/32")
 
-# Define a custom instance lister for cluster/log-dump/log-dump.sh.
-function log_dump_custom_get_instances() {
```

Comment:
I _think_ this is safe, as long as we have a sufficiently new kubetest.  But we're close to being able to get rid of kops-e2e-runner entirely, which would then be safer.

Reply:
@justinsb this makes my local testing a lot easier as well and will speed up testing times.  We have installing aws cli running around in multiple places, unfortunately.